CONTRIBUTING.md

```diff
@@ -14,11 +14,11 @@
 
 #### **Do you want to add a new feature?**
 
-- [Open a new issue](https://github.com/TheAlgorithms/Java/issues/new). Be sure to include a **title and a clear description** and a **test case** demonstrating the new feature that you want to add to the project.
+- [Open a new issue](https://github.com/TheAlgorithms/Java/issues/new). Be sure to include a **title, clear description** and **test case** demonstrating the new feature you want to add to the project.
```

Comment:
```suggestion
- [Open a new issue](https://github.com/TheAlgorithms/Java/issues/new). Be sure to include a **title**, **clear description**, and a **test case** demonstrating the new feature that you want to add to the project.
```
Why remove `that` and `a`? They are there for a reason

Reply:
**a** is already mentioned before title, so we don't need to mention after every comma elements in the sentence this prevents plagiarism in the grammer so I had removed **a** after **and*.

**that** is not required in this statement, the sentence is self-explanatory, so I had removed it to make the paragraph clean and concise.